src/containers/SubjectPage/SubjectContainer.jsx

```diff
@@ -197,6 +198,12 @@ const SubjectPage = ({
     }	
   };
 
+  const onClickTopics = (e, subTopics) => {
+    e.preventDefault();
+    const target = subTopics.find(t => t.label === e.currentTarget.textContent)
+    history.replace(target.url)
```

Comment:
Det å matche tekst-label mot hverandre er ikke helt sikkert. Man kan ikke utelukke at at noen har like navn. Det trengs ikke slåes opp i subTopics da man har url på elementet.
```suggestion
    history.replace(e.currentTarget.href);
```